core/platform/users/gae_current_user_services.py

```diff
@@ -53,33 +49,13 @@ def is_current_user_super_admin():
     return users.is_current_user_admin()
 
 
-def get_gae_id_from_email(email):
-    """Given an email address, returns a gae id.
-
-    Returns None if the email address does not correspond to a valid user id.
-    """
-    class _FakeUser(ndb.Model):
-        """A fake user class."""
-
-        _use_memcache = False
-        _use_cache = False
-        user = ndb.UserProperty(required=True)
-
+def get_user_by_email(email):
```

Comment:
Can this function be entirely removed from here, if it's not being used by the non-test code at all?

Reply:
Yup, done.